iceoryx_hoofs/source/cxx/helplets.cpp

```diff
@@ -21,27 +21,33 @@ namespace iox
 {
 namespace cxx
 {
+// AXIVION DISABLE STYLE AutosarC++19_03-A5.2.4 : allow reinterpret_cast for the whole file
```

Comment:
Does this mean that the rule is suppressed for the whole file? If so, I'm not sure if I like this. It is correct for now and facilitates readability but this file could be extended in the future. The line that enables the rule again could be easily overlooked and a new function might be added after this line with an unnecessary/wrong used  reinterpret_cast and the tool would not be aware of this.

Reply:
@FerdinandSpitzschnueffler The rule is suppressed LOC between `DISABLE` and `ENABLE` cooment

Reply:
Okay. As I mentioned above it may be dangerous to suppress the rule for the whole file but it seems you changed this in https://github.com/eclipse-iceoryx/iceoryx/pull/1395.